components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -448,7 +455,7 @@ - (void)updateTrailingViewAlpha {
 - (CGFloat)trailingViewAlpha {
   // The trailing view has the same behavior as .rightView in UITextField: It has visual precedence
   // over the clear button.
-  CGFloat trailingViewAlpha = self.trailingView.alpha;
```

Comment:
The warning is wrong here. If self.trailingViewMode is garbage (not one of the enums), then trailingViewAlpha is an arbitrary value. The original code is safer.